rmgpy/molecule/molecule.py

```diff
@@ -1618,4 +1618,21 @@ def toGroup(self):
         
         return group
 
-
+    def getNthNeighbor(self, startingAtoms, n, ignoreList = []):
```

Comment:
One possible way to allow a list of distances:
```
def getNthNeighbor(self, startingAtoms, distances, ignoreList=[], n=1):
    `distances` is a list of integers, corresponding to desired neighbor distances
```
Then most of the rest stays the same, until the last part:
```
    if n < max(distances):
        if n in distances:
            neighbors += self.getNthNeighbor(neighbors, distances, ignoreList, n+1)
        else:
            neighbors = self.getNthNeighbor(neighbors, distances, ignoreList, n+1)

    return neighbors
```
I haven't tested this, but I think it should work. Feel free to ignore this if you prefer the current method (or if it doesn't work like I think it does).